static/css/authentication.css

```diff
@@ -0,0 +1,196 @@
+/*!
+ * Licensed under the MIT license - http://opensource.org/licenses/MIT
+ * @author BhaumikPatel
+ * @site https://bootsnipp.com/snippets/featured/full-page-sign-in
+ */
+
+.my_container
```

Comment:
Containers, base colors and a lot of general stuff should be separated in base.css file, this fild should be first <link on head section